scene/2d/canvas_item.cpp

```diff
@@ -337,7 +337,6 @@ void CanvasItem::_update_callback() {
 		notification(NOTIFICATION_DRAW);
 		emit_signal(SceneStringNames::get_singleton()->draw);
 		if (get_script_instance()) {
-			Variant::CallError err;
```

Comment:
I'm not sure those are actually unused. IIRC I wanted to remove them a year or two ago and was told not to.

Reply:
Namely https://github.com/godotengine/godot/issues/5568#issuecomment-231007723

Reply:
I'll have a look at the constructor again then, but it looked pretty pointless to me.

Reply:
It has the default constructor and can't do anything. There's no way this can affect anything.

```c++
	struct CallError {
		enum Error {
			CALL_OK,
			CALL_ERROR_INVALID_METHOD,
			CALL_ERROR_INVALID_ARGUMENT,
			CALL_ERROR_TOO_MANY_ARGUMENTS,
			CALL_ERROR_TOO_FEW_ARGUMENTS,
			CALL_ERROR_INSTANCE_IS_NULL,
		};
		Error error;
		int argument;
		Type expected;
	};
```

Reply:
I wonder if `call_multilevel()` once returned an error (like `call()` does) and it was changed for some reason leaving behind some `CallError` variables. In any case, there's no error checking code, so this variable is indeed unused.